vm/rust/src/lib.rs

```diff
@@ -421,16 +422,60 @@ fn build_block_context(
 }
 
 
+
```

Comment:
Oops, 😹 cat jumped on <enter> here ;)

Reply:
Will remove it later, don't want to block release